parameters.d/osg36-el9.yaml

```
@@ -0,0 +1,60 @@
+###################
+# OSG 3.6 tests
+# File format documention:
+# https://github.com/opensciencegrid/vm-test-runs#running-osg-test-in-vm-universe
+###################
+
+platforms:
+  - centos_stream_9_x86_64
+  - rocky_9_x86_64
+  - alma_9_x86_64
+
+sources:
+  ###################
+  # Format:
+  # [<Github account>:<osg-test branch>;] <OSG ver>; <REPO 1, REPO 2...REPO N> [> <Update OSG ver>/<Update REPO 1, REPO 2...REPO N>]
+  # Example:
+  # Run osg-test (from 3.2-minefield) with packages from 3.2-release and 3.2-testing that are then upgraded to
+  # 3.3-testing and 3-3-upcoming-testing:
+  # 3.2; osg, osg-testing > 3.3/osg-testing, osg-upcoming-testing
+  ###################
+  - opensciencegrid:master; 3.6; osg
+  - opensciencegrid:master; 3.6; osg, epel-testing
+  - opensciencegrid:master; 3.6; osg-testing
+  - opensciencegrid:master; 3.6; osg > osg-testing
+  # - opensciencegrid:master; 3.5; osg, osg-upcoming > 3.6/osg
+  # - opensciencegrid:master; 3.6; osg, osg-upcoming
+  # - opensciencegrid:master; 3.6; osg-testing, osg-upcoming-testing
+  # - opensciencegrid:master; 3.6; osg > osg-testing, osg-upcoming-testing
+  # - opensciencegrid:master; 3.5; osg, osg-upcoming > 3.6/osg-minefield
+  # - opensciencegrid:master; 3.6; osg-minefield
+  # - opensciencegrid:master; 3.6; osg > osg-minefield
```

```suggestion
  # - opensciencegrid:master; 3.6; osg, osg-upcoming
  # - opensciencegrid:master; 3.6; osg-testing, osg-upcoming-testing
  # - opensciencegrid:master; 3.6; osg > osg-testing, osg-upcoming-testing
  - opensciencegrid:master; 3.6; osg-minefield
  - opensciencegrid:master; 3.6; osg > osg-minefield
```
3.5 will never support EL9. Also we should uncomment the minefield tests for now until we have more stuff in testing.